vacuum-advisor/vacuum-simulator.mdx

```diff
@@ -0,0 +1,182 @@
+---
+title: 'VACUUM Simulator in Schema Statistics'
+backlink_href: /docs/vacuum-advisor
+backlink_title: 'VACUUM Advisor'
+---
+
+VACUUM Simulator lets you tweak autovacuum settings to learn the relation
+between settings and autovacuum patterns.
+It also helps you find better autovacuum settings for the table.
+
+## Overview
+
+Autovacuum can be triggered by the following 3 causes:
+
+1. Triggered by dead rows
+2. Triggered by freeze age
+3. Triggered by inserts
+
+Each cause has related configuration settings. By tweaking these settings, you
+can see changes in the simulation data.
+There is a panel for each cause, and each panel has the actual/current value and
+graph section to show how it was in the selected time range, and simulation
+value and graph section to show how it is based on the collected statistics and
+the adjustable configuration settings.
+You can also create a permalink of your current simulation and modified settings
+to share with your team members.
+
+### Limitation
+
+* The current autovacuum count can be much bigger than the simulation count with
+  frequently autovacuumed tables, since the simulation can only perform
+  autovacuum every 10 minutes.
```

Comment:
Uhmm yeah, I understand your point, though I don't have good alternative.

I think people can still use simulation for these tables to:

* understand what is causing so many vacuums
* find out how they can reduce autovacuum count

How does mentioning these sound?

Reply:
Yeah, that sounds reasonable. Maybe something like
```
* The current autovacuum count can be much bigger than the simulation count with
  frequently autovacuumed tables, since the simulation can only perform
  autovacuum every 10 minutes. The simulation can still be useful to understand why
  so many vacuums are performed, and to explore ways to reduce autovacuum frequency.
```
?